resources/js/Components/CampaignJoinModal/CampaignJoinModal.tsx

```diff
@@ -38,11 +36,10 @@ type CampaignJoinModalDataType = {
 }
 
 const CampaignJoinModal = ({open, onClose, message}: CampaignJoinModalPropType) => {
-    const {data, setData, errors, clearErrors, post} = useForm<CampaignJoinModalDataType>(
-        {
-            code: undefined,
-        },
-    )
+    const {t} = useTranslation()
+    const {data, setData, errors, clearErrors, get} = useForm<CampaignJoinModalDataType>({
```

Comment:
honestly don't think we need `errors` here, it's unlikely we get an error from this, or have you tried entering wrong code?
also i don't think entering an invite link works here either, should it be working?